pkg/kubelet/hosts_test_file1

```diff
@@ -0,0 +1,8 @@
+# hosts file for testing.
```

Comment:
i think it's brittle to try to write files during a test. what if the test run fails unexpectedly? then the file stays there and you end up potentially polluting the testing infrastructure :/